src/command/graph/push.rs

```diff
@@ -43,8 +45,15 @@ impl Push {
             &self.profile_name
         );
 
-        let schema_document = get_schema_from_file_path(&self.schema_path)
-            .context("Failed while loading from SDL file")?;
+        let schema_document = if &self.schema == "-" {
+            let mut buffer = String::new();
+            io::stdin().read_to_string(&mut buffer)?;
+            buffer
+        } else {
+            get_schema_from_file_path(&self.schema).context("Failed while loading from SDL file")?
+        };
+
+        tracing::debug!("Schema Document to push:\n{}", &schema_document);
```

Comment:
our debug traces should take advantage of `tracing`'s structured log functionality instead of just strings so we can further interact with the type if necessary

```suggestion
        tracing::debug!(schema_document, "pushing schema document");
```